menu.php

```diff
@@ -0,0 +1,155 @@
+        "Sides",
+        array("French Fries", "", "2"),
+        array("Duck Fat Tater Tots", "", "2")
+    );
+    $beverages = array (
+        "Beverages",
+        array("Bottled Soda & Water", "", "1.50"),
+        array("Iced Tea", "", "1"),
+        array("Coffee & Hot Tea", "", "1")
+    );
+    $desserts = array (
+        "Desserts",
+        array("Cookies", "", "1"),
+        array("Bars/Fruit Bread", "", "1.50"),
+        array("Special Dessert", "", "2.50"),
+        array("Thimmie's Bread Pudding", "", "2.50")
+    );
+    $lunch = array($entrees, $sandwiches, $salads, $sides, $beverages, $desserts);
+
+    $starters = array (
+        "Starters",
+        array("Charcuterie", "chef's selection house cured | meats & local cheeses | black garlic aioli | raisin whole grain mostarda | blackberry | radish | bread" , "11"),
+        array("Beets", "balsamic | mission figs | orange | pistachio | honey vinaigrette", "6"),
+        array("Butternut Soup", "farro | chestnuts | black truffle | spiced rye crouton", "7"),
+        array("Roasted Marrow", "chermoula | maitake mushroom | cipollini onion | parsley | lemon | rustic bread", "8")
+    );
+    $special_entrees = array (
+        "Special Entrees",
+        array("Short Rib", "bordelaise | yukon potato puree | peas | cipollini onion | celery | beach mushrooms", "18"),
+        array("Rabbit Ragout", "cavatelli | maitake mushroom | broccoli | cured yolk | parmesan | fresh tarragon" , "15"),
+        array("Duck Breast", "whole grain raisin mostarda | parsnip puree | heirloom carrot | trumpet mushrooms | cipollini onion | blackberry", "19"),
+        array("Salt Cured Salmon", "sweet corn cream | tart apple & celery root salad | quinoa | onion | picled chilies | poppy | cilantro", "17")
+    );
+    $special = array($starters, $special_entrees);
+    $menu = array($regular_entrees, $sandwiches, $salads, $sides, $beverages, $desserts, $starters, $special_entrees);
+
+    function generatePage($array) {
+        $title = $array[0];
+        $id = str_replace(' ', '', $array[0]);
+        unset($array[0]);
+        echo "    <div data-role=\"page\" data-title=\""; 
+        echo $title; 
+        echo "\" id=\""; 
+        echo $id; 
+        echo "\">
+        <header data-role=\"header\" data-add-back-btn=\"true\" data-position=\"fixed\">
+            <h1>"; 
+        echo $title; 
+        echo "</h1>
+        </header>
+        <div data-role=\"content\">
+            <ul data-role=\"listview\" data-inset=\"true\">";
+        foreach ($array as $item) {
+            echo "<li><h3>";
+            echo $item[0];
+            echo "</h3><p><i>";
+            echo $item[1];
+            echo "</i></p><span class=\"ui-li-count\">";
+            echo "$" . $item[2];
+            echo "</span></li>";
+        }
+        echo "</ul>
+        </div>
+        <footer style=\"text-align: center;\" data-role=\"footer\" data-position=\"fixed\">MCC Sage Bistro</footer>
+    </div>";
+    }
+?>
```

Comment:
Aren't you fancy! You could easily adapt this to scrape values out of a text file or off a form to allow the chefs to update their own menu. Very nice!